transport/grpc/config.go

```diff
@@ -151,6 +153,11 @@ func (c InboundTLSConfig) newInboundCredentials() (credentials.TransportCredenti
 //        tls:
 //          enabled: true
 //        compressor: gzip
+//        grpc-keepalive:
```

Comment:
What is the reason of having the prefix `grpc-`? We could remove it and only have keepalive since we are in the gRPC scope already no?

Reply:
I intentionally added the `grpc` prefix to make it very clear that this is grpc application layer keepalive and not the TCP keepalive setting, as gRPC outbound can have custom TCP keepalive as well.